as2/server/redemption.md

```diff
@@ -0,0 +1,16 @@
+# Redemption
+##### Messages pertaining to the redemption server.
+---
+### Join Server
+Sent to the penguin trying to log in to the redemption server to give needed player information. 
+
+`rjs`
+
+|Argument|Type|Variadic?
```

Comment:
It seems this table is missing some formatting characters (the ending pipe or | characters).